src/PaymentLink.php

```diff
@@ -5,20 +5,22 @@
 use Requests;
 
 
-class Payment_link extends Entity
+class PaymentLink extends Entity
 {
     /**
      * Creates Payment link .
      *
      * @param array $attributes
      *
-     * @return Payment_link
+     * @return PaymentLink
      */
     public function create($attributes = array())
     {
-        $additionHeader['Content-Type'] = 'application/json';
+        $attributes = json_encode($attributes);
 
-        parent::setAdditionHeader($additionHeader) ;
+        $request = new Request;
+        
+        $request->addHeader('Content-Type', 'application/json');
```

Comment:
addHeader is static method can be used directly `Request::addHeader('Content-Type', 'application/json');`

Reply:
Yes we can, corrected